_apidocs/regulationsgov/v4/openapi.yaml

```diff
@@ -293,31 +285,32 @@ paths:
             application/vnd.api+json:
               schema:
                 $ref: '#/components/schemas/JSONError'
-    post:
-      summary: Creates a new comment.
+  /comments/{documentId}:
+    get:
+      summary: Get detailed information for specified documentId
+      description: Gets the detailed information of a specific document with the passed documentId.
       tags:
         - comments
       parameters:
         - name: documentId
           in: path
-          description: ID of document the comment is being submitted to
+          description: ID of document to return
           required: true
           schema:
             type: string
-      requestBody:
-        description: A JSON object containing comment information
-        required: true
-        content:
-          application/vnd.api+json:
-            schema:
-                $ref: '#/components/schemas/JSONResourcePostRequestObject'
+        - name: include #Only supported value for include is attachments.
+          in: query
+          description: resources to include
+          example: attachments
+          schema:
+            type: string
       responses:
-        '201':
-          description: Created
+        '200':
+          description: successful operation
           content:
             application/vnd.api+json:
               schema:
-                $ref: '#/components/schemas/JSONResourcePostResponseObject'
+                $ref: '#/components/schemas/CommentFindOneResponse'
```

Comment:
There were no functional updates above. I moved POST comment under /comments. Earlier the post method was under /comments/{documentId}. I also deleted documentId parameter from under our POST /comments call. Our POST /comments only takes a requestBody, no parameters now.